tests/helpers/perf_run_helper.rb

```diff
@@ -97,9 +98,38 @@ def create_perf_archive_root
     @gplt_timestamp = Time.now.getutc.to_i
     @archive_root = "#{PERF_RESULTS_DIR}/PERF_#{@gplt_timestamp}"
     FileUtils.mkdir_p @archive_root
```

Comment:
It seems like the line:
```
FileUtils.mkdir_p @archive_root
```
is no longer necessary because:
```
FileUtils.mkdir_p "#{@archive_root}/hosts"
```
will create it as well.

Reply:
yeah, I tried merging them... but the resulting changes to the spec tests made them less clear.  Like I couldn't have a spec test to ensure the archive root alone was created, which is a critical dep for other things... Because the call would be for hosts not just archive root... So kinda have to choose, clearer spec test names and such, or one less call the mkdir_p.  Thoughts?  I could really go either way on it.

Reply:
That makes sense, thanks. Could you please add a comment to the code explaining that so someone reading / refactoring in the future doesn't get this same idea?

Reply:
added